toolkit/scripts/sodiff/mariner-sodiff.sh

```diff
@@ -1,12 +1,37 @@
 #!/bin/bash
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+sodiff_script_error=false
+while getopts "r:f:v:o:e:" opt; do
```

Comment:
Since we're here, I'm curious if we couldn't also re-name this to `azurelinux-sodiff.sh` or something along those lines. Just a P2 suggestion, though.